package.json

```diff
@@ -93,11 +93,11 @@
     "mocha": "^3.5.3",
     "redux": "^4.0.0",
     "rimraf": "^2.5.4",
-    "rxjs": "^6.0.0",
+    "rxjs": "^7.0.0",
     "sinon": "^4.5.0",
     "ts-loader": "^6.2.0",
-    "tslib": "^1.10.0",
-    "typescript": "^3.6.4",
+    "tslib": "~2.1.0",
+    "typescript": "~4.2.2",
     "webpack": "^4.29.3",
```

Comment:
Should we look at updating webpack to 5?

Reply:
Perhaps, but out of scope for this PR